docs/client/nakama/relay.mdx

```diff
@@ -148,3 +148,59 @@ catch (ApiResponseException ex)
     Debug.LogFormat("Error: {0}", ex.Message);
 }
 ```
+
+### Viewing Receipts and Events
+
+Game clients can view [events](/cardinal/game/system/event) emitted in systems as well as the results of iterating over [messages](/cardinal/game/message#iterating-over-messages) by listening for notifications from Nakama.
+
+The code to receive notifications will be different depending on the Nakama client you are using. [Client specific instructions can be found here](https://heroiclabs.com/docs/nakama/concepts/notifications/#receive-notifications)
```

Comment:
I'd move this under `Example` and rephrase it like this.

> The code to receive notifications will be different depending on the game engine/client you are using. Learn more about how to receive notifications for your game client at [Nakama's docs](https://heroiclabs.com/docs/nakama/concepts/notifications/#receive-notifications)

Reply:
done